.buildkite/scripts/pipelines/security_solution_quality_gate/api_integration/api-integration-tests.sh

```diff
@@ -18,12 +18,29 @@ QA_API_KEY=$(vault_get security-solution-quality-gate qa_api_key)
 
 # Generate a random 5-digit number
 random_number=$((10000 + $RANDOM % 90000))
-ENVIRONMENT_DETAILS=$(curl --location 'https://global.qa.cld.elstc.co/api/v1/serverless/projects/security' \
+if [ -z "${KIBANA_MKI_USE_LATEST_COMMIT+x}" ] || [ "$KIBANA_MKI_USE_LATEST_COMMIT" = "0" ]; then
+  ENVIRONMENT_DETAILS=$(curl --location 'https://global.qa.cld.elstc.co/api/v1/serverless/projects/security' \
     --header "Authorization: ApiKey $QA_API_KEY" \
     --header 'Content-Type: application/json' \
     --data '{
-        "name": "ftr-integration-tests-'$random_number'",
-        "region_id": "aws-eu-west-1"}' | jq '.')
+          "name": "ftr-integration-tests-'$random_number'",
```

Comment:
Why are we using a random number?

Reply:
This way it works both against buildkite or locally. Otherwise I should have to do if else statements in bash script to use an alternative for each case of the above. When this will be moved into the normal runner such as parallel_serverless, they can fix it.